bart/sched/pelt.py

```diff
@@ -0,0 +1,347 @@
+#    Copyright 2015-2016 ARM Limited
+#    Copyright 2016 Google Inc. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+PELT Simulation Model
+"""
+
+from collections import namedtuple as _namedtuple
+from pandas import DataFrame as _DataFrame
```

Comment:
This is normal for python, it has nothing to do with ipython.  Take any module that you import and do a dir().  You will probably see most of the modules it imports.

In the example that you use, 

``` python
peltsim = pelt.Simulator(...)
peltsim.<TAB>
```

In this case, it will autocomplete for methods defined in the Simulator class, not methods defined in the `pelt` module, so you won't have any problems.

The example you were looking for is:

``` python
from bart.sched import pelt
pelt.<TAB>
```

That would autocomplete imported stuff, like for most python modules.  If that really bothers you there are ways of controlling what is available there, but the `_` is not the right solution.